quaddtype/numpy_quaddtype/src/ops.hpp

```diff
@@ -13,13 +13,13 @@ typedef Sleef_quad (*unary_op_quad_def)(const Sleef_quad *);
 // Unary Quad operations with 2 outputs (for modf, frexp)
 typedef void (*unary_op_2out_quad_def)(const Sleef_quad *, Sleef_quad *, Sleef_quad *);
 
-static Sleef_quad
+[[maybe_unused]] static Sleef_quad
 quad_negative(const Sleef_quad *op)
 {
     return Sleef_negq1(*op);
 }
 
-static Sleef_quad
+[[maybe_unused]] static Sleef_quad
```

Comment:
Why not make it static inline like all others then?

Reply:
Done